client/dashboard/sites/scan/index.tsx

```diff
@@ -138,7 +143,16 @@ function SiteScan( { scanTab }: { scanTab: 'active' | 'history' } ) {
 						}
 					/>
 				}
-				notices={ <ScanNotices status={ status } threatCount={ threatCount } /> }
+				notices={
+					<VStack as="div" spacing={ 3 }>
```

Comment:
I know this is copied from site logs code, but `as="div"` is the default value; we can just remove it :)